modules/insticatorBidAdapter.js

```diff
@@ -190,17 +205,59 @@ function buildDevice(bidRequest) {
   return device;
 }
 
+function _getCoppa() {
+  return config.getConfig('coppa') === true ? 1 : 0;
+}
+
+function _getGppConsent(bidderRequest) {
+  let gpp = deepAccess(bidderRequest, 'gppConsent.gppString')
```

Comment:
I thought all privacy policy values were available by default from prebid js core module. Why do we need to rely on the publisher's bidderRequest object?

Reply:
No, it comes from the bidderRequest. 
This is how all other adaptors are set up as well.

Reply:
@rohanInsticator Check this https://docs.prebid.org/dev-docs/bidder-adaptor.html#bidderrequest-parameters